Tests/SwiftDocCTests/Test Bundles/ObjCFrameworkWithInvalidLink.docc/Info.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FIY: You don't need an Info.plist unless you're specifying default availability or default syntax highlighting. You can remove this file and the test will work the same.

Reply:
removed